docs/source/ingest/destination_connectors/code/python/opensearch.py

```diff
@@ -0,0 +1,62 @@
+import os
+
+from unstructured.ingest.connector.elasticsearch import (
+    ElasticsearchWriteConfig,
+)
+from unstructured.ingest.connector.local import SimpleLocalConfig
+from unstructured.ingest.connector.opensearch import (
+    OpenSearchAccessConfig,
+    SimpleOpenSearchConfig,
+)
+from unstructured.ingest.interfaces import (
+    ChunkingConfig,
+    EmbeddingConfig,
+    PartitionConfig,
+    ProcessorConfig,
+    ReadConfig,
+)
+from unstructured.ingest.runner import LocalRunner
+from unstructured.ingest.runner.writers.base_writer import Writer
+from unstructured.ingest.runner.writers.opensearch import (
+    OpenSearchWriter,
+)
+
+
+def get_writer() -> Writer:
+    return OpenSearchWriter(
+        connector_config=SimpleOpenSearchConfig(
+            access_config=OpenSearchAccessConfig(
+                hosts=os.getenv("OPENSEARCH_HOSTS"),
+                username=os.getenv("OPENSEARCH_USERNAME"),
+                password=os.getenv("OPENSEARCH_PASSWORD"),
+            ),
+            index_name=os.getenv("OPENSEARCH_INDEX_NAME"),
+        ),
+        write_config=ElasticsearchWriteConfig(
+            batch_size_bytes=15_000_000,
```

Comment:
why is batch in bytes rather than number of records?

Reply:
@ahmetmeleq Above question. Why did we choose bytes instead of records?

Reply:
Basically due to using bulk requests (for performance) and how elasticsearch nodes process bulk requests:
https://www.elastic.co/guide/en/elasticsearch/guide/current/indexing-performance.html#_using_and_sizing_bulk_requests
![image](https://github.com/Unstructured-IO/unstructured/assets/39141206/ae844b51-2793-4c9c-8ae4-914c7274b63d)